src/lib/form/schemas/transfers.ts

```diff
@@ -17,7 +19,7 @@ type CrossChainTransferFormData = {
 };
 
 type CrossChainTransferValidationParams = {
-  [CROSS_CHAIN_TRANSFER_AMOUNT_FIELD]: FeesValidationParams & MaxAmountValidationParams & MinAmountValidationParams;
```

Comment:
Fees validation needs to be handled separately so removing for now—will follow in a separate PR.

Reply:
Fees validation is out of scope for now—will be implemented when it's handled across the application.